src/main/java/com/google/sps/data/Challenge.java

```diff
@@ -12,7 +12,7 @@
 
   private Type challenge_type;
   private String name;
-  private List <String> steps;
+  private List <Pair<String,String>> steps;
```

Comment:
nit: Add a comment explaining what the pair of strings are supposed to represent.